### PR TITLE
Fix GCP stacks which aren't building.

### PR DESCRIPTION
**Pull request by jlewi**
* The GCP stack doesn't build (kubeflow/gcp-blueprints#55). The problem
  is pipeline GCP manifest is referring to a configmap that doesn't
  exist

  * This is a temporary hack to make the manifests build; I just commented
    out the missing file.

  * @bobgy will hopefully add an appropriate long term fix.

* The GCP stack was reintroduced in #1286. The unittests should have
  caught this but they didn't because make generate is swallowing errors
  if there is a problem generating the tests #1269

Update the alice example to use the GCP config

### CODE REVIEW

**Comment by jlewi** [comment]
/assign @Bobgy 
/assign @animeshsingh 
/assign @Jeffwan 

@Jeffwan or @animeshsingh Would one of you mind reviewing this? (@Bobgy is in China and its Friday night) and I'd like to get this merged ASAP to fix  GCP master.

**Comment by Bobgy** [comment]
I can actually take a quick look

File: pipeline/mysql/installs/gcp-pd/kustomization.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ resources:
 - ../../../upstream/env/platform-agnostic/mysql
 - ../../overlays/application/application.yaml
 - persistent-volume.yaml
-- ../generic/configmap.yaml
+# TODO(https://github.com/kubeflow/gcp-blueprints/issues/55): This configmap doesn't exist; jlewi commented
+# it out so the manifests would build but we need to figure out the correct long term fix.
+# - ../generic/configmap.yaml
```
Comment (Bobgy) [inline pipeline/mysql/installs/gcp-pd/kustomization.yaml:13]: Sorry, this is a mistake I forgot to remove after refactoring

Comment (Bobgy) [reply pipeline/mysql/installs/gcp-pd/kustomization.yaml:13]: /lgtm
Do you want to remove the comment too?

**Comment by jlewi** [comment]
@Bobgy I'll remove it in a follow on PR because I'd like this to get merged.

/approve

**Pull request merged**